tests/smoke-test/jest.config.mjs

```diff
@@ -0,0 +1,5 @@
+import config from '../../jest.config.mjs';
+export default {
+  ...config,
```

Comment:
the config here should be same as the e2e-tests. The global setup/teardown should not be called. Same for the tsconfig.

Reply:
What do you mean by "Same for the tsconfig."

Reply:
I meant the additional testconfig.test.json that we added for unit testing:

The transform option should be:
```
transform: {
    '^.+\\.tsx?$': [
      'ts-jest',
      {
        useESM: true,
      },
    ],
  },
```

and not (from the common jest.config.mjs)
```
transform: {
    '^.+\\.tsx?$': [
      'ts-jest',
      {
        useESM: true,
        tsconfig: path.resolve(__dirname, 'tsconfig.test.json'),
      },
    ],
  }
  ```